### DISCUSSION TITLE
List of widths ignored

### DISCUSSION THREAD

**Discussion by npgarcia** [percy/percy-selenium-java#193] [Q&A]
I am trying to take a screenshot in a specific width, but when I look at the screenshots, all the screenshots in the configuration are stored. 

That specific test is only for that width (desktop); the rest of the widths in the configuration are considered mobile. This is the code to take the screenshot. Is the parameter being ignored? Or the configuration overwrites the code?

```
Percy percy = new Percy(driver); 
percy.snapshot(name, Arrays.asList(1200));

**Comment by itsjwala** [comment]
hey @npgarcia 👋 

* is this an issue anymore? 
* if so can you share a build to debug further?
